packages/core/utils/src/prettyDiagnostic.js

```diff
@@ -32,16 +41,21 @@ export default function prettyDiagnostic(
       : [codeFrame.codeHighlights];
 
     let formattedCodeFrame = formatCodeFrame(codeFrame.code, highlights, {
-      useColor: true
+      useColor: true,
+      syntaxHighlighting: true,
+      language:
+        // $FlowFixMe sketchy null checks do not matter here...
+        language || filePath ? path.extname(filePath).substr(1) : undefined
```

Comment:
I think you meant `language || (filePath ? path.extname(filePath).substr(1) : undefined)` ?

Reply:
ow yeah